### PR TITLE
完成所有内容的翻译

### PR DESCRIPTION
**Pull request by RisingInIris2017**
补完了之前没有做完的竹匾内容。
剩下的方块-物品内容也全部翻译完毕。

有两处翻译疑点：
1. 生命汲取效果中，汲取生命值的数量。
https://github.com/RisingInIris2017/After-the-Drizzle-Document-EN/blob/master/effects/life-drain.md
公式计算得到的是心的数量还是生命值点数？
生命汲取 3 汲取的生命值是 1.25 心还是 2.5 心？
2. 光合作用效果中，回复生命值的数量。问题同上。

### CODE REVIEW

File: blocks-items/bamboo-tray.md [hunk 1]
```diff
@@ -57,26 +57,31 @@ and the Bamboo Tray is not in Bake Mode, Process Mode, and In-rain Mode,
...
-将竹匾放置在石制[弹射板](catapult-board.md)上，可装备成自动化竹匾。
+## Automated Bamboo Tray
 
-在附近有红石信号激活的情况下， 当竹匾内物品完成一次加工时，会将竹匾内的物品朝弹射板方向弹出一格。
+Put Bamboo Tray on Stone [Catapult Board](catapult-board.md) will automate it.
 
-可以利用这个特点，在该方向放置漏斗进行收集。
+If Stone Catapult Board is redstone-activated, every time the Bamboo Tray finishes one recipe,
+
+the product will be capculted 1 block far, towards the direction which the Catapult Board face to.
```
Comment (RoShioLeo) [inline blocks-items/bamboo-tray.md:85]: "capculted" should be "catapulted".

File: blocks-items/bamboo-tray.md [hunk 2]
```diff
@@ -57,26 +57,31 @@ and the Bamboo Tray is not in Bake Mode, Process Mode, and In-rain Mode,
...
+The amount of time fermentation takes is decided by the Humid of that area.
 
-### ![](../.gitbook/assets/descriptions/bake_mode.png) 烘焙模式
+### ![](../.gitbook/assets/descriptions/bake_mode.png) Bake Mode
 
-将竹匾放在点燃的炉灶上方即可切换到烘焙模式。
+Put Bamboo Tray on a lit Stove, and it will be switched to Bake Mode.
 
-### ![](../.gitbook/assets/descriptions/process_mode.png) 加工模式（暂时无用）
+### ![](../.gitbook/assets/descriptions/process_mode.png) Process Mode(WIP)
```
Comment (RoShioLeo) [inline blocks-items/bamboo-tray.md:66]: Add a space before punctuation.

File: blocks-items/drink-maker.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
-## 交互
+## How to craft
 
-使用可以装入流体的物品右击沏茶台，可以倒入或倒出沏茶台内的流体。
\ No newline at end of file
+![Any type of Planks * 3 + Sticks * 2 → Drink Maker * 1](../.gitbook/assets/recipes/drink_maker_recipe.png)
+
+## How to interact
+
+Holding a container which is able to contain fluids(Vanilla Buckets, for example), and right-clicking the Drink Maker,
```
Comment (RoShioLeo) [inline blocks-items/drink-maker.md:20]: I think "fluid container" may be simple enough with following examples.
Just like this:
Holding a fluid container (Vanilla Buckets, for example)...

File: blocks-items/filter-screen.md [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
+## How to craft
 
-![木棍 * 4 + 线 * 4 + 红石粉 * 1 → 滤网 * 1](../.gitbook/assets/recipes/filter_screen_recipe.png)
+![Sticks * 4 + Strings * 4 + Redstone Dust * 1 → Filter Screen * 1](../.gitbook/assets/recipes/filter_screen_recipe.png)
 
-## 使用
+## How to use
 
-- 使用空手右击滤网可使滤网改变方向。
+- Right-click the Filter Screen with an empty hand to change direction it faced to.
```
Comment (RoShioLeo) [inline blocks-items/filter-screen.md:15]: Faced or facing? I'm not sure, either.

File: blocks-items/filter-screen.md [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
 
-- 在滤网上方放置可存储物品的容器，并在其内放置物品，即可让该物品的实体通过滤网而不受阻拦。多个滤网在竖直方向上叠加时，都会以上方最近的容器作为白名单列表。
+- Put a container block on the Filter Screen, and put some items into that container block, then the filter will stop any item entities but the items which is in that container from going through it.
```
Comment (RoShioLeo) [inline blocks-items/filter-screen.md:17]: then the filter will stop any item entities except items in the container.
Or
then they can freely go through the filter.

File: blocks-items/gauze.md [hunk 6]
```diff
@@ -1,16 +1,15 @@
-# 纱 Gauze
+# Gauze
 
-**纱（Gauze）**
-是模组加入的一种可用在护甲上防暑的物品。
+**Gauze** is an item, which you can craft with armor and provide those pieces of armor with heat protection function.
```
Comment (RoShioLeo) [inline blocks-items/gauze.md:3]: Please use "Heat Resistance". It has been used in the language file.

File: blocks-items/handwarmer.md [hunk 7]
```diff
@@ -1,24 +1,25 @@
...
+## How to craft
 
-![铁锭 * 5 + 铁粒 * 1 → 手炉 * 1](../.gitbook/assets/recipes/handwarmer_recipe.png)
+![Iron Ingots * 5 + Iron Nugget * 1 → Handwarmer * 1](../.gitbook/assets/recipes/handwarmer_recipe.png)
 
-## 使用
+## How to use
 
-- 暖和的手炉可提供2点防寒值，需手持（含副手）才可生效。
+- Warm Handwarmer provides the holder 2 points of cold-resistant, only when it is held in either hands.
```
Comment (RoShioLeo) [inline blocks-items/handwarmer.md:13]: en_us.lang uses "Hot Handwarmer".
If necessary, it can be changed to "Warm".

File: blocks-items/hole.md [hunk 8]
```diff
@@ -1,12 +1,11 @@
-# 坑 Hole
+# Hole
 
-**坑（Hole）**
-是自然生成的一种方块，挖掘可获得物品。
+**Hole** is a block generated naturally. Mining them will get some items.
```
Comment (RoShioLeo) [inline blocks-items/hole.md:3]: Digging! It's just dirt.

File: blocks-items/hole.md [hunk 9]
```diff
@@ -1,12 +1,11 @@
...
 
-## 说明
+## What is it
 
-坑有很小概率在野外生成。
+Holes generated in the wild with tiny amount of probability.
 
-由于目前仅能挖掘到竹子，故仅在湿度为湿润及以上的地区生成。
\ No newline at end of file
+Since it only contains Bamboo now, it is generated in areas where Humid is above Moderate.
```
Comment (RoShioLeo) [inline blocks-items/hole.md:11]: In the humidity system, "湿润" uses "Moist" as its English name.

File: blocks-items/insulating-layer.md [hunk 10]
```diff
@@ -1,16 +1,14 @@
-# 防寒层 Insulating Layer
+# Insulating Layer
 
-**防寒层（Insulating Layer）**
-是模组加入的一种可用在护甲上御寒的物品。
+**Insulating Layer** is an item, which you can craft with armor and provide those pieces of armor with cold protection function.
```
Comment (RoShioLeo) [inline blocks-items/insulating-layer.md:3]: Cold Resistance

File: blocks-items/porcelain-teapot.md [hunk 11]
```diff
@@ -1,22 +1,21 @@
...
+## How to craft
 
-![粘土 * 3 → 粘土壶坯 * 1](../.gitbook/assets/recipes/clay_teapot_recipe.png)
+![Clay Blocks * 3 → Clay Teapot * 1](../.gitbook/assets/recipes/clay_teapot_recipe.png)
 
-## 用途与特点
+## How to use
 
-- 右键流体可以装入壶中（岩浆等流体除外），可以放置。
+- Right-clicking fluid block will contain that fluid into the Teapot, except Lava. Then you can place the Teapot on the ground.
```
Comment (RoShioLeo) [inline blocks-items/porcelain-teapot.md:13]: Then?

File: blocks-items/tea-leaves.md [hunk 12]
```diff
@@ -1,22 +1,25 @@
...
 
-**茶叶（Tea Leaves）**
-是由茶树获得的一种叶子及其加工产品，目前总共有三类茶。
+**Tea Leaves** is literally leaves of Tea Plants, and processed tea leaves used for tea-making.
 
-## 各类茶叶
+There are 3 variants of Tea Leaves at present.
 
-![现有的几种茶叶](../.gitbook/assets/blocks-items/tea_leaves.png)
+## Variants of Tea Leaves
```
Comment (RoShioLeo) [inline blocks-items/tea-leaves.md:9]: Use the confirmed English name.

File: blocks-items/tea-plant.md [hunk 13]
```diff
@@ -1,27 +1,29 @@
...
+![Wild Tea Plants](../.gitbook/assets/blocks-items/wild_tea_plant.png)
 
-野生茶树会自然生成于湿度级别大于等于湿润的生物群系。
+## Wild Tea Plants
 
-打掉会掉落2~4个茶叶和1~2个茶籽。
+Wild Tea Plants generated in biomes whose Humid is above Moderate.
 
-## 生长习性
+Mining them will get 2-4 Tea Leaves and 1-2 Tea Tea Seeds.
```
Comment (RoShioLeo) [inline blocks-items/tea-plant.md:15]: Why do you like to use "mine"?
Use breaking or destroying.

File: features/crops.md [hunk 14]
```diff
@@ -1,17 +1,24 @@
-# 作物环境系统
+# Crop Environment System
 
-**作物环境系统（Crop Environment System）**
-是模组根据[湿度](humid.md#湿度)系统而引入的农业系统。
+**Crop Environment System** is a mechanics added by the *After the Drizzle* Mod,
 
-## 说明
+which is based on [湿度](humid.md#Humid) system, and influences your farming gameplay.
```
Comment (RoShioLeo) [inline features/crops.md:5]: 湿度……

File: features/crops.md [hunk 15]
```diff
@@ -1,17 +1,24 @@
...
 
-目前所有作物（不包含其他模组果树），均会有一个适宜湿度；如果该作物不在支持列表内，则会给予默认适宜湿度：一般~湿润。
+## Description
 
-当作物在该湿度环境下生长时，与原版无差异。超出该范围临界值每一级，生长概率下降50%。
+All crops(except trees added by other mods) get a proper humid level. 
 
-同时模组默认禁用使用骨粉对作物催熟，催生草方块与促使茶树开花等功能仍正常。
+If a certain type of crop is not included in the support list, it gets a default proper humid level: Moderate to Moist.
```
Comment (RoShioLeo) [inline features/crops.md:11]: “Moderate” → “Average”

**Comment by RoShioLeo** [comment]
全部审阅完成，请按照批注修正。
有关括号格式问题仅提了一次。

**Comment by RoShioLeo** [comment]
另外英文文档维护请署名

File: blocks-items/bamboo-tray.md (hunk 1 above)
Comment (RisingInIris2017) [reply blocks-items/bamboo-tray.md:85]: Has been fixed in the latest commit.

File: blocks-items/bamboo-tray.md (hunk 2 above)
Comment (RisingInIris2017) [reply blocks-items/bamboo-tray.md:66]: Has been fixed in the latest commit.

File: blocks-items/filter-screen.md (hunk 5 above)
Comment (RisingInIris2017) [reply blocks-items/filter-screen.md:17]: Thanks for providing better translation.
I think the former is better, for the word "they" means those items in the container, not item entities.

File: blocks-items/filter-screen.md (hunk 4 above)
Comment (RisingInIris2017) [reply blocks-items/filter-screen.md:15]: maybe "it faces to" is better, replacing the phrase to a clause.

File: blocks-items/gauze.md (hunk 6 above)
Comment (RisingInIris2017) [reply blocks-items/gauze.md:3]: Has  been fixed in the latest commit.

File: blocks-items/handwarmer.md (hunk 7 above)
Comment (RisingInIris2017) [reply blocks-items/handwarmer.md:13]: Fixed.

File: blocks-items/hole.md (hunk 8 above)
Comment (RisingInIris2017) [reply blocks-items/hole.md:3]: Fixed.

File: blocks-items/hole.md (hunk 9 above)
Comment (RisingInIris2017) [reply blocks-items/hole.md:11]: In the Chinese version of this doc, it says "湿度为湿润及以上".
The phrase "above Moderate" excludes Moderate level itself, and express the same meaning IMO.

File: blocks-items/insulating-layer.md (hunk 10 above)
Comment (RisingInIris2017) [reply blocks-items/insulating-layer.md:3]: Fixed.

File: blocks-items/tea-leaves.md (hunk 12 above)
Comment (RisingInIris2017) [reply blocks-items/tea-leaves.md:9]: Fixed.

File: blocks-items/porcelain-teapot.md (hunk 11 above)
Comment (RisingInIris2017) [reply blocks-items/porcelain-teapot.md:13]: I misunderstood the original texts.
Fixed.

File: blocks-items/hole.md (hunk 9 above)
Comment (RoShioLeo) [reply blocks-items/hole.md:11]: 我的意思是，应该统一为我湿度系统已经定义的英文名……

File: blocks-items/tea-plant.md (hunk 13 above)
Comment (RisingInIris2017) [reply blocks-items/tea-plant.md:15]: I found a better translation.
Fixed.

File: blocks-items/hole.md (hunk 9 above)
Comment (RoShioLeo) [reply blocks-items/hole.md:11]: 那应该是Average

File: blocks-items/porcelain-teapot.md (hunk 11 above)
Comment (RoShioLeo) [reply blocks-items/porcelain-teapot.md:13]: 我感觉中文确实会有点歧义。

File: features/crops.md (hunk 15 above)
Comment (RisingInIris2017) [reply features/crops.md:11]: I read the https://github.com/RoitoLeonine/After-the-Drizzle-Document/blob/master/features/humid.md again.
There is no word "Average" existing. Do you want to use this word?
If you indeed think this word is better, I will change all "Moderate" to "Average" to keep the consistency.

Comment (RoShioLeo) [reply features/crops.md:11]: The document was wrong. I correct it.
In the game, Humid never uses "Moderate".
I won't use the same word in Rainfall and Humid.

**Pull request merged**